src/settings.js

```diff
@@ -337,7 +337,7 @@ var EMULATE_FUNCTION_POINTER_CASTS = false;
 // [link]
 var EXCEPTION_DEBUG = false;
 
-// If 1, export `demangle` and `stackTrace` helper function.
+// If 1, export `demangle` and `stackTrace` JS library functions.
```

Comment:
Is this comment still accurate? If so, then are we exporting a legacy function using a non-legacy setting?

Reply:
True... I would like to make this settings a legacy.  But if I move it to LEGACY_SETTINGS it cannot be read.   I'll mark is as `[deprecated]` instead.

Reply:
Done